packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -413,6 +443,23 @@ export interface IEnvironment extends IResource {
    */
   readonly environmentArn: string;
 
+  /**
+   * Creates a deployment of the supplied configuration to this environment.
+   * Note that you can only deploy one configuration at a time to an environment.
+   * However, you can deploy one configuration each to different environments at the same time.
+   * If more than one deployment is requested for this environment, they will occur in the same order they were provided.
+   *
+   * @param configuration The configuration that will be deployed to this environment.
+   */
+  addDeployment(configuration: IConfiguration): void;
+
+  /**
+   * Creates a deployment for each of the supplied configurations to this environment.
```

Comment:
I'm assuming the order that the configurations are specified in the array matters here, if that's correct could you add a sentence to explain that?